src/section-negative-numbers.mbx

```diff
@@ -321,9 +321,9 @@
         </figure>
 
         <p>This illustrates that <m>-3\cdot(-7)=21</m>, and it would seem that a negative number times a negative number always gives a positive result.</p>
-
```

Comment:
It's fine to remove this paragraph, but then I think there should be something to replace it. Something that discusses multiplication by 0. But maybe that belongs in a different section?